packages/nextjs-mf/src/internal.ts

```diff
@@ -93,40 +93,36 @@ export const DEFAULT_SHARE_SCOPE: SharedObject = {
 };
```

Comment:
The function 'parseRemotes' can be simplified by using a switch statement instead of multiple if statements. This can improve readability and maintainability.